tests/testthat/test-get_code.R

```diff
@@ -266,6 +266,19 @@ testthat::test_that("@linksto tag indicate affected object if object is assigned
 
 # @linksto ---------------------------------------------------------------------------------------------------------
 
+testthat::test_that("get_code does not break if @linksto is with eval in last line", {
```

Comment:
Does this cover only `eval` or other functions as well?

Reply:
Other as well. I put this solution first to fix `foo()` example from `get_code` docs

Reply:
However we can rename the test to "get_code does not break if @linksto is in the last line"

Reply:
Once this is merged to #263 I will do a thorough tests cleanup in there